### PR TITLE
adding PRODUCT DEFINITION TEMPLATE 4.49

### LINKED ISSUE #132: Adding GRIB2 template 4.49 for GEFSv13 aerosol ensemble use.
[GRIB2 table 4.49](https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_temp4-49.shtml) is missing in g2tmpl source code.
GEFSv13 ensemble aerosol products can not be correctly generated.

### LINKED ISSUE #135: Add template 4.49
To encode ensemble member information in aerosol fields, the [template 4.49](https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_temp4-49.shtml) should be added in grib2_all_tables_module.F90.

### PR DESCRIPTION
**Pull request by edwardhartnett**
Fixes #132 
Fixes #135

### CODE REVIEW

**Review by Hang-Lei-NOAA — APPROVED** [review]

File: src/grib2_all_tables_module.F90
Comment (lipan-NOAA) [inline src/grib2_all_tables_module.F90:1]: looks good to me.

**Review by lipan-NOAA — APPROVED** [review]

**Review by lipan-NOAA — APPROVED** [review]

File: src/grib2_all_tables_module.F90
Comment (edwardhartnett) [reply src/grib2_all_tables_module.F90:1]: Do you have any example files that use this?

Can you point me to the code in UPP which will be using this template?

**Comment by WenMeng-NOAA** [comment]
@edwardhartnett Please also add @AndrewBenjamin-NOAA as a reviewer.

File: src/grib2_all_tables_module.F90
Comment (lipan-NOAA) [reply src/grib2_all_tables_module.F90:1]: /lfs/h2/emc/ptmp/Li.Pan/o/ep5/com/gefs.20180816/00/p02/gfs.20180816/00/model_data/atmos/history

/lfs/h2/emc/ens/noscrub/li.pan/global-workflow/sorc/ufs_model.fd/FV3/upp

**Pull request merged**